### PR TITLE
add fitness option to `pose-to-viz.py`

### PR DESCRIPTION
**Pull request by JenkeScheen**
quick PR that satisfies a requirement for https://github.com/choderalab/asapdiscovery/issues/566, now have a CLI endpoint to generate either a subpocket or fitness HTML starting from a PDB file and an SDF.

### CODE REVIEW

**Comment by JenkeScheen** [comment]
for that PR we'll also still need to augment `pose-to-viz-pymol` so that we can roll out .pse files that also have the fitness viz - some complexities there that are out of scope for this PR.

File: asapdiscovery-data/asapdiscovery/data/fitness.py [hunk 1]
```diff
@@ -133,8 +133,10 @@ def parse_fitness_json(target: TargetTags) -> pd.DataFrame:
             f"Specified target is not valid, must be one of: {TargetTags.get_values()}"
         )
 
-    if target == "MERS-CoV-Mpro":
-        raise NotImplementedError("MERS-CoV-Mpro fitness data not yet available.")
+    if target not in ("SARS-CoV-2-Mpro", "SARS-CoV-2-Mac1"):
```
Comment (hmacdope) [inline asapdiscovery-data/asapdiscovery/data/fitness.py:136]: I have some code centralising this WIP, that I can transfer to this PR or I can do post-hoc.

**Review by hmacdope — APPROVED** [review]
Looks good! See my comment on issue about unifying they pymol and non-pymol version of the scripts but that's kind of seperate.

File: asapdiscovery-data/asapdiscovery/data/fitness.py (hunk 1 above)
Comment (JenkeScheen) [reply asapdiscovery-data/asapdiscovery/data/fitness.py:136]: post-hoc is fine, cheers

**Pull request merged**